app/views/adjustments/new.html.erb

```diff
@@ -1,15 +1,15 @@
 <section class="content-header">
 <% content_for :title, "#{current_organization.name} - Inventory - Adjustments - New" %>
   <h1>
-    New Adjustment
+    New Inventory Adjustment
     <small>for <%= current_organization.name %></small>
   </h1>
   <ol class="breadcrumb">
   <li><%= link_to(dashboard_path) do %>
       <i class="fa fa-dashboard"></i> Home
     <% end %>
     </li>
-    <li><%= link_to "Distributions", (adjustments_path) %></li>  
+    <li><%= link_to "Distributions", (adjustments_path) %></li>
```

Comment:
I think we have this mislabeled. Shouldn't it be:
`<li><%= link_to "Inventory Adjustments", (adjustments_path) %></li>`
If we are linking to the `adjustments_path`?

Reply:
Yes you are probably correct. I'll double check